kvk/Issues/official/description.xml

```diff
@@ -19,27 +19,237 @@
 		<nl>https://privacybydesign.foundation/attribute-index/nl/irma-demo.kvk.official.html</nl>
 	</IssueURL>
 	<Attributes>
-		<Attribute id="legalEntity">
+		<Attribute id="peilmoment">
```

Comment:
In credentials of other issues we often use English attribute identifiers. You might consider doing that here too to such that similar attribute types (i.e. emailAddress) have the same identifier in most credentials.